sip/sip-006-runtime-cost-assessment.md

```diff
@@ -887,78 +888,149 @@ Usage:
 (contract-call? .cost-voting-contract "get-proposal" 123)
 ```
 
-### Voting for a Cost-Assessment Proposal
+### Voting
+
+#### Stacks Holders
+
+Stacks holders can vote for cost-assessment function proposals by calling the
+**Clarity Cost Voting Contract's** `vote` function. The `vote` function takes
+two arguments, `proposal-id` and `amount`. `proposal-id` is the ID of the proposal
+being voted for. `amount` is the amount of STX the voter would like to vote *with*.
+The amount of STX you include is the number of votes you are casting.
```

Comment:
What if this rather is quadratic, that is, the amount of STX you include is the square of the number of votes you're casting?